tests/unittests/caffe2ImporterTest.cpp

```diff
@@ -1255,3 +1255,52 @@ TEST(caffe2, EQ1D) {
   // Graph has two inputs and one output.
   EXPECT_EQ(mod.getPlaceholders().size(), 3);
 }
+
+// Test loading a LengthsToRanges operator.
+TEST(caffe2, LengthsToRanges) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetDescFilename(
+      "tests/models/caffe2Models/lengths_to_ranges.pbtxt");
+  std::string NetWeightFilename(
+      "tests/models/caffe2Models/lengths_to_ranges_init_net.pbtxt");
+
+  Placeholder *output;
+  Context ctx;
+
+  // Destroy the loader after the graph is loaded
+  {
+    Caffe2ModelLoader caffe2LD(NetDescFilename, NetWeightFilename, {}, {}, *F);
+    output = caffe2LD.getSingleOutput();
+    ctx.allocate(mod.getPlaceholders());
+  }
+
+  // High level checks on the content of the graph.
+  // We have 1 LengthsToRanges and 1 Save.
+  EXPECT_EQ(F->getNodes().size(), 2);
+
+  // Check that the graph has the expected shape (LengthsToRanges -> Save),
+  // starting from the output.
+  auto *saveNode = getSaveNodeFromDest(output);
+  auto *N = llvm::dyn_cast<LengthsToRangesNode>(saveNode->getInput());
+  ASSERT_TRUE(N);
+
+  // Graph has one output.
+  EXPECT_EQ(mod.getPlaceholders().size(), 1);
+
+  // Compile and run the model.
+  auto *res = ctx.get(output);
+  EE.compile(CompilationMode::Infer, F, ctx);
+  EE.run();
+
+  // Check that the output tensor is the same as the expected output.
+  // auto result = res->getHandle<int64_t>();
+  Tensor expected(ElemKind::Int64ITy, {4, 2});
```

Comment:
that might be debatable, but I think the idea of loader tests is to test the loader, e.g., check that loaded graph contains certain Glow nodes.
this is not always possible and then we do some run checks, I think here it's fine just to check structure.

Reply:
Funny you should point this out :-).  I was in fact specifically interested in testing execution, because of the int32_t vs. int64_t issue; I was concerned that if we were somehow reading the init_net as int32, we could do the wrong computation here.  (I think that's impossible, given we don't even have the machinery to read int32 tensors :) ).

Reply:
I'll go ahead and kill the execution stuff, anyways.  It probably doesn't make sense given how things work now.